### ISSUE TITLE
BUG: Address label is not displaying Country Name in International petitioner contact info

### ISSUE THREAD

**Issue by ttlenard** [ustaxcourt/ef-cms#8303]
**Describe the Bug**
If a Petitioner has paper service, and the petitioner has an international address, the address coversheet that is generated on the Print for paper service screen is not including the full address. The Country information is missing.  This applies to any address label page for any served filings.

Question - are there any other fields missing from the contact information that are not being displayed on the address label?

Not sure if this info is helpful or not?  It was helpful for me when I was poking around to find the original story for the address label page.
Here is the original story for adding in the address label page: ustaxcourt/ef-cms#6957.
Here is a related bug that I found where the address label was missing the c/o info: ustaxcourt/ef-cms#7315

**Business Impact/Reason for Severity**
Medium severity - Court may not be mailing to the appropriate address - could be considered ineffective service

**In which environment did you see this bug?**
Production, Test

**Who were you logged in as?**
Docket Clerk

**What were you doing when you discovered this bug? (Using the application, demoing, smoke tests, testing other functionality, etc.)**
Reported to DAWSON team by the Court.

**To Reproduce**
1. As a docket clerk, go to a case
2. Click on Case information, and then click on the parties tab
3. Click the edit button for a petitioner that has paper service
4. Update the Country to International (if it's not already)
5. Add in some information for a new updated address, including the Country name
6. Click save
7. Next, QC the Notice of change document on the docket record
8. On the print for paper service screen, notice that the country is missing from the address label.

**Expected Behavior**
Whatever address is input into DAWSON should display on the Address Coversheet for paper service

**Actual Behavior**
A clear and concise description of what actually happened.
An International address is not displaying correctly.

**Screenshots**

Here is the screen capture of the address cover sheet and the Order 
![image](https://user-images.githubusercontent.com/87717140/197866388-9ecdec97-c371-4e24-b539-b54c118e3b90.png)

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Cause of Bug, If Known**

**Process for Logging a Bug:**
* Complete the above information
* Add a severity tag (Critical, High Severity, Medium Severity or Low Severity). See below for priority definition. 

**Severity Definition:**
* Critical Defect
Blocks entire system's or module’s functionality
No workarounds available
Testing cannot proceed further without bug being fixed.

* High-severity Defect
Affects key functionality of an application
There's a workaround, but not obvious or easy
App behaves in a way that is strongly different from the one stated in the requirements

* Medium-severity Defect
A minor function does not behave in a way stated in the requirements.
Workaround is available and easy

* Low-severity Defect
Mostly related to an application’s UI
Doesn't need a workaround, because it doesn't impact functionality

## Definition of Ready for Bugs(Created 10-4-21)
Definition used: A failure or flaw in the system which produces an incorrect or undesired result that deviates from the expected result or behavior. (Note: Expected results are use cases that have been documented in past user stories as acceptance criteria and test cases, and do not include strange behavior unrelated to use cases.)

The following criteria must be met in order for the development team to begin work on the bug.

The bug must:
 - Be focused on solving a user problem
 - Contain data for all fields in the bug template, so the team can pick it up and begin working immediately

Process: If the unexpected results are new use cases that have been identified, but not yet built, new acceptance criteria and test cases should be captured in a new user story and prioritized by the product owner. 

If the Court is not able to reproduce the bug, add the “Unable to reproduce” tag. This will provide visibility into the type of support that may be needed by the Court. In the event that the Court cannot reproduce the bug, the Court will work with Flexion to communicate what type of troubleshooting help may be needed.

## Definition of Done (Updated 4-14-21)
**Product Owner**
 - [ ]  Bug fix has been validated in the Court's test environment

**Engineering**
 - [ ] Automated test scripts have been written
 - [ ] Field level and page level validation errors (front-end and server-side) integrated and functioning
 - [ ] Verify that language for docket record for internal users and external users is identical
 - [ ] New screens have been added to pa11y scripts
 - [ ] All new functionality verified to work with keyboard and macOS voiceover https://www.apple.com/voiceover/info/guide/_1124.html 
 - [ ] READMEs, other appropriate docs, JSDocs and swagger/APIs fully updated
 - [ ] UI should be touch optimized and responsive for external only (functions on supported mobile devices and optimized for screen sizes as required)
 - [ ] Interactors should validate entities before calling persistence methods
 - [ ] Code refactored for clarity and to remove any known technical debt
 - [ ] Deployed to the Court's `test` environment if prod-like data is required. Otherwise, deployed to any `experimental` environment for review.

**Comment by ttlenard** [comment]
Hi @rosiehsmith Testing Feedback for 10/27/2022.
Country info is displayed on the address label now for the print for paper service screen for when Docket serves something on the Docket record. Thanks!  Unfortunately, I did find a related issue with paper petitions when they get served.  See below:

I decided to do some additional checking for when a Paper Petition comes in.  It looks like the Country is missing on the address label spot on the Notice of Receipt of Petition as well.  I'm sorry I didn't catch this earlier!  Steps to recreate:

1. Log in as a Petitions Clerk
2. Click on Document QC
3. Click Start a new petition
4. Fill out all the information required, and be sure that the petitioner has an international address
5. Save and Serve to the IRS
6. Notice that on the Print Paper Petition receipt, the Country is also missing here.

![image.png](https://images.zenhubusercontent.com/60f99b8ac0d8d04c8aed1aa3/40ddfd48-320c-46cf-b4fc-29236481fa71)

Question: Should I enter a new bug, or shall it be handled in this one?
